src/load/azext_load/data_plane/utils/validators.py

```diff
@@ -228,12 +228,26 @@ def validate_test_plan_path(namespace):
     namespace.test_plan = _validate_path(namespace.test_plan, is_dir=False)
 
     _, file_extension = os.path.splitext(namespace.test_plan)
-    if file_extension.casefold() != ".jmx":
+    if file_extension.casefold() != ".jmx" and file_extension.casefold() != ".json":
         raise InvalidArgumentValueError(
             f"Invalid test plan file extension: {file_extension}. Expected: .jmx"
```

Comment:
Need to update the error message as well.

Reply:
updated